portfolio/src/main/webapp/script.js

```diff
@@ -16,7 +16,13 @@
  * Adds a random greeting to the page.
  */
 function addFunFact() {
-    const funFacts = ['Jennifer Almost Didn\'t Return For The Final Season',
+
+    //to fetch quote from the servlet and add it to our home page with a heading using ARROW function
+    fetch('/random-quote').then(response => response.text()).then( (quote)=> {
+        document.getElementById('fun-fact-container').innerText= '\n Fun fact about friends 101: \n' + quote;});
```

Comment:
Be consistent how you use arrow functions: Either use parenthesis around single-arguments or don't. Just don't mix them. Line breaks could be better. There are spaces missing before `=>` and `=`.

This is what our formatter does:

```
  fetch('/random-quote').then((response) => response.text()).then((quote) => {
    document.getElementById('fun-fact-container').innerText =
        '\n Fun fact about friends 101: \n' + quote;
  });
```

Reply:
Hi Steffen, could you tell me where can I find this formatter? Where should I download it from and will it work with the cloud shell editor?

Reply:
In Cloud Shell, you can probably install a linter as command line tool, for example https://www.npmjs.com/package/jslint. I don't think there's a way to include the linter in the IDE, but I have not worked with it yet.

In VSCode you can install packages which automatically format your files when you save.